### DISCUSSION TITLE
实验室里的导入数据一直不成功

### DISCUSSION THREAD

**Discussion by dottjaoo** [WatsonYao/nian_app#52] [Q&A] [answered]
试了好多种格式都提示“未提取到数据”，能给个导入文件的模板吗，我想直接在里面改试试能不能导入成功。手机是小米10，系统MIUI 14.0.4

**Comment by WatsonYao** [comment, marked as answer]
[import.txt](https://github.com/WatsonYao/nian_app/files/12839689/import.txt)
请使用系统app或者直接用电脑把 import.txt 拷贝到 nian 文件夹，部分第三方软件拷贝的话，有无法读写的权限问题。

**Reply by dottjaoo** [reply]
可以用了非常感谢。我试出来为什么我的识别不出来了，之前一直以为是别的地方格式错了，原来是因为我日期写的不标准，我7月直接输入“7”了，但是要“07”才能识别😂。不好意思，因为这种问题打扰你🙏。

**Reply by WatsonYao** [reply]
感谢你的反馈，可以帮助到更多用户。
